### PR TITLE
Updating the venue of the conference to Shaheed Sukhdev College of Business Studies

### PR DESCRIPTION
**Pull request by emht**
Issue #106 
@rajataaron, @realslimshanky

### CODE REVIEW

**Comment by realslimshanky** [comment]
Hi @emht ,
Please vertical alignment of the venue text so that it looks even with the date.
Also to make sure that both date + venue div are at the center please make the venue multi line as follows,
`Shaheed Sukhdev College`
`of Business Studies`
`Delhi, NCR`
Look for options which ever make them look good.
image

File: index.html [hunk 1]
```diff
@@ -125,7 +125,8 @@ <h1 class="animated zoomIn">PyCon India 2017</h1>
                   </li>
                   <li>
                     <i class="fa fa-location-arrow"></i>
-                    <span>Delhi-NCR</span>
+                    <!-- Changing the venue from "Delhi, NCR" to "SSCBS" -->
+                    <span align="center"><a style="color:#FFF;" href="https://www.google.co.in/maps/place/Shaheed+Sukhdev+College+of+Business+Studies" target="_blank"> Shaheed Sukhdev College<br /> Of Bussiness Studies</a></span>
```
Comment (realslimshanky) [inline index.html]: You can simply use `<br>` instead of `<br />`.

File: templates/index.html [hunk 2]
```diff
@@ -36,7 +36,8 @@ <h1 class="animated zoomIn">PyCon India 2017</h1>
                   </li>
                   <li>
                     <i class="fa fa-location-arrow"></i>
-                    <span>Delhi-NCR</span>
+                    <!-- Changing the venue from "Delhi, NCR" to "SSCBS" -->
+                    <span align="center"><a style="color:#FFF;" href="https://www.google.co.in/maps/place/Shaheed+Sukhdev+College+of+Business+Studies" target="_blank"> Shaheed Sukhdev College<br /> Of Bussiness Studies</a></span>
```
Comment (realslimshanky) [inline templates/index.html]: You can simply use `<br>` instead of `<br />`.

**Review by realslimshanky — CHANGES_REQUESTED** [review]

File: templates/index.html (hunk 2 above)
Comment (emht) [reply templates/index.html]: Ok, on it.

Comment (emht) [reply templates/index.html]: Done @realslimshanky

**Review by realslimshanky — APPROVED** [review]
LGTM

File: index.html [hunk 3]
```diff
@@ -125,7 +125,8 @@ <h1 class="animated zoomIn">PyCon India 2017</h1>
                   </li>
                   <li>
                     <i class="fa fa-location-arrow"></i>
-                    <span>Delhi-NCR</span>
+                    <!-- Changing the venue from "Delhi, NCR" to "SSCBS" -->
+                    <span align="center"><a style="color:#FFF;" href="https://www.google.co.in/maps/place/Shaheed+Sukhdev+College+of+Business+Studies" target="_blank"> Shaheed Sukhdev College<br>Of Bussiness Studies</a></span>
```
Comment (realslimshanky) [inline index.html]: Please correct the spelling, 'Bussiness' to 'Business'.

File: templates/index.html [hunk 4]
```diff
@@ -36,7 +36,8 @@ <h1 class="animated zoomIn">PyCon India 2017</h1>
                   </li>
                   <li>
                     <i class="fa fa-location-arrow"></i>
-                    <span>Delhi-NCR</span>
+                    <!-- Changing the venue from "Delhi, NCR" to "SSCBS" -->
+                    <span align="center"><a style="color:#FFF;" href="https://www.google.co.in/maps/place/Shaheed+Sukhdev+College+of+Business+Studies" target="_blank"> Shaheed Sukhdev College<br>Of Bussiness Studies</a></span>
```
Comment (realslimshanky) [inline templates/index.html]: Please correct the spelling of 'Bussiness' to 'Business'.

**Review by realslimshanky — CHANGES_REQUESTED** [review]

**Pull request merged**